CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Release Notes
 
+## 0.1.0 (TBD)
```

Comment:
😍 no longer alpha!

Reply:
will be deployed when 
- rio-tiler gets to 2.0
- cogeo-mosaic gets to 3.0
😄

Reply:
aka 2022? 😉